components/Header/index.js

```diff
@@ -9,4 +9,31 @@
 //  </div >
 // And add it to the DOM in the .header-container component
 
-function Header() {}
+
+
+function header() {
+    const 
+        headerCont = document.querySelector('.header-container'),
+        header = document.createElement('div'),
+        date = document.createElement('span'),
+        h1 = document.createElement('h1'),
+        temp = document.createElement('span');
+
+    header.classList.add('header');
+    date.classList.add('date');
+    temp.classList.add('temp');
+
+    header.appendChild(date);
+    header.appendChild(h1);
+    header.appendChild(temp);
+
+    date.textContent = 'March 28, 2019';
+    h1.textContent = 'Lambda Times';
+    temp.textContent = '98°';
+
+    headerCont.appendChild(header);
```

Comment:
Is it time to append? remember your still inside the function and the function needs to 'do/return' only one thing that interacts with other aspects of your program